components/pages/home/questAndCollectionTabs.tsx

```diff
@@ -231,7 +231,7 @@ const QuestAndCollectionTabs: FunctionComponent<
             )}
           </CustomTabPanel>
           <CustomTabPanel value={tabIndex} index={1}>
-            <div className="flex flex-col items-center space-y-6">
+            <div className="flex flex-col items-center space-y-6 pb-5 lg:pb-0">
```

Comment:
This should not be added there, but on line 214 (the quest list)

Reply:
I moved the code to the line you said.